eng/scripts/Language-Settings.ps1

```diff
@@ -9,6 +9,21 @@ $PackageRepositoryUri = "https://pypi.org/project"
 ."$PSScriptRoot/docs/Docs-ToC.ps1"
 ."$PSScriptRoot/docs/Docs-Onboarding.ps1"
 
+function Get-python-AdditionalValidationPackagesFromPackageSet {
+  param(
+    [Parameter(Mandatory=$true)]
+    $LocatedPackages,
+    [Parameter(Mandatory=$true)]
+    $diffObj
+  )
+
+  $additionalValidationPackages = @()
+
+  Write-Host "We are in Get-python-AdditionalValidationPackagesFromPackageSet getting called."
```

Comment:
I assume this is to be filled in later.

Reply:
I've spent the bulk of today dealing with a new python livesite. This should have already been done.

Reply:
Ok got it populated now.